lib/libc/picolibc/CMakeLists.txt

```diff
@@ -11,6 +11,10 @@ zephyr_library_sources(
   stdio.c
   )
 zephyr_include_directories(include)
+if(CONFIG_PICOLIBC_USE_MODULE)
+  # temporary workaround for https://github.com/picolibc/picolibc/pull/1079
```

Comment:
a) Is this related to this commit?
b) In any case, shouldn't a bug be filled to follow up on this and remove it when it is not needed anymore?

Reply:
> A tracking bug within the Zephyr Project can definitely be created and closed whenever the fix is merged into the SDK or Picolibc module

Please do, so we remember to remove this workaround then.